samples/openapi3/client/petstore/typescript/builds/default/apis/PetApi.ts

```diff
@@ -17,14 +17,14 @@ export class PetApiRequestFactory extends BaseAPIRequestFactory {
 
     /**
      * Add a new pet to the store
-     * @param pet Pet object that needs to be added to the store
+     * @param Pet Pet object that needs to be added to the store
      */
-    public async addPet(pet: Pet, _options?: Configuration): Promise<RequestContext> {
+    public async addPet(Pet: Pet, _options?: Configuration): Promise<RequestContext> {
```

Comment:
is this an intentional change? or rather - is it really required for the proposed fix?